test/python/lattice_tight_binding.py

```diff
@@ -16,11 +16,16 @@
 # Authors: Nils Wentzell
 
 import unittest
+import sys
+
 from triqs.lattice import BravaisLattice, BrillouinZone, TightBinding
 from triqs.lattice.tight_binding import TBLattice
 from h5 import HDFArchive
 
 import numpy as np
+sys.path.append('/triqs-sympyfy/python/triqs/lattice/')
```

Comment:
I think this is specific to your triqs install right? Please remove

Reply:
The reason why I used `sys.path` was that I wanted to import the `utils.py` file into the unit test so that I could use `TB_to_sympy_2D`. After looking up how to do so, I found that I could use an absolute path to the file using `sys.path`. At the time, I did not think that would pose a problem since utils.py exists in the same folder on your (TRIQS's) end too, so it would work for both of us.
If there is another way I could import `utils.py`, please tell me so. I would appreciate your help.

Reply:
I removed it.